### PR TITLE
feat(ci/build): run test-ramdisk in build-windows workflow

### PR DESCRIPTION
**Pull request by NaddiNadja**

### CODE REVIEW

**Review by karlowich — APPROVED** [review]
Nice!

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -220,6 +220,7 @@ jobs:
       uses: actions/download-artifact@v4.1.7
       with:
         name: xnvme-src-archive-with-subprojects
+
```
Comment (safl) [inline .github/workflows/verify.yml:228]: This re-format, could it be moved into it's own commit? Or is it essential for the feature:
```
feat(ci/build): run test-ramdisk in build-windows workflow 
```
?

**Review by safl — APPROVED** [review]
This is excellent, I have a comment inline, but not a blocker.

File: .github/workflows/verify.yml (hunk 1 above)
Comment (NaddiNadja) [reply .github/workflows/verify.yml:228]: Not essential, i have moved it to another commit now :)

**Review by safl — APPROVED** [review]
LGTM.

**Review by karlowich — APPROVED** [review]
